packages/odyssey-react-mui/scripts/properties-to-json.js

```diff
@@ -0,0 +1,100 @@
+/*!
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+const { resolve, join, basename, extname } = require("node:path");
+const properties = require("properties");
+const {
+  ensureDirSync,
+  ensureFileSync,
+  readFileSync,
+  writeFileSync,
+  removeSync,
+} = require("fs-extra");
+const readdir = require("recursive-readdir");
+
+const convert = (baseFiles, propertiesTargetDir) => {
+  for (const src of baseFiles) {
+    const filename = basename(src);
+    const extension = extname(src);
+    const source = `${readFileSync(src)}`;
+
+    properties.parse(source, function (error, obj) {
+      // eslint-disable-next-line no-console
+      if (error) {
+        return console.error(error);
+      }
+
+      const targetFile = join(
+        propertiesTargetDir,
+        filename.replace(extension, ".ts")
+      );
+      ensureFileSync(targetFile);
+      writeFileSync(
+        targetFile,
+        "/* eslint-disable header/header */\nexport const translation = " +
+          JSON.stringify(obj) +
+          ";"
+      );
+    });
+  }
+};
+
+async function convertPropertiesToJson({ resourcePath, targetJsonPath }) {
+  const sourceDirectory = resolve(resourcePath);
+  const propertiesTargetDirectory = resolve(targetJsonPath);
+
+  ensureDirSync(sourceDirectory);
+  removeSync(propertiesTargetDirectory);
+  ensureDirSync(propertiesTargetDirectory);
+
+  let baseFiles = await readdir(sourceDirectory);
+  convert(baseFiles, propertiesTargetDirectory);
+}
+
+exports.handler = async (argv) => {
+  const { resourcePath, targetJsonPath } = argv;
+
+  await convertPropertiesToJson({
+    resourcePath,
+    targetJsonPath,
+  });
+};
```

Comment:
This was taken from globalization-core

Reply:
Let's put a comment at the top of the file and link to the file in `globalization-core`. That way future devs will know where to look.

Reply:
This works? @KevinGhadyani-Okta 
```
// Part of this has been copied over from @okta/ui-build-tools' own internal node scrip
// https://github.com/okta/ui-build-tools/blob/master/packages/clis/i18n/properties-to-json.js
```